Makefile

```diff
@@ -58,7 +58,10 @@ binaries: clean \
 	_output/share/lima/lima-guestagent.Linux-x86_64 \
 	_output/share/lima/lima-guestagent.Linux-aarch64 \
 	_output/share/lima/lima-guestagent.Linux-armv7l \
-	_output/share/lima/lima-guestagent.Linux-riscv64
+	_output/share/lima/lima-guestagent.Linux-riscv64 \
+	_output/share/lima/lima-guestagent.FreeBSD-x86_64 \
+	_output/share/lima/lima-guestagent.FreeBSD-aarch64 \
+	_output/share/lima/lima-guestagent.FreeBSD-riscv64
```

Comment:
Eventually we may want to have `configure` or `make menuconfig` to specify build targets

Reply:
* https://github.com/lima-vm/lima/pull/1790